src/AppBundle/Command/CheckFrameworkVersionCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\HealthCheck;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use Psr\Log\LoggerInterface;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CheckFrameworkVersionCommand extends Command
+{
+
+    protected $doctrine;
+
+    protected $logger;
+
+    /**
+     * @param \Symfony\Bridge\Doctrine\RegistryInterface $doctrine
+     * @param \Psr\Log\LoggerInterface $logger
+     */
+    public function __construct(RegistryInterface $doctrine, LoggerInterface $logger)
+    {
+        $this->doctrine = $doctrine;
+        $this->logger = $logger;
+        parent::__construct(null);
+    }
+
+    public function configure()
+    {
+        $this
+          ->setName('app:check-version')
+          ->setDescription('Checks the current version of a CMS or Framework.')
+        ;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $healthChecks = $this->doctrine->getRepository(HealthCheck::class)->findAll();
+
+        /** @var HealthCheck $healthCheck */
+        foreach ($healthChecks as $healthCheck) {
+            $res = $this->checkVersion($healthCheck->getUrl());
+            if ($res) {
+                $healthCheck->getSite()->setFrameworkVersion($res['version']);
+                $healthCheck->setLastSyncAt(new \DateTime());
+            }
+        }
+
+        $this->doctrine->getEntityManager()->flush();
+        $this->doctrine->getEntityManager()->clear();
+    }
+
+    /**
+     * @param $url
+     *
+     * @return bool|mixed
+     */
+    protected function checkVersion($url)
```

Comment:
Lets make it return just version, as a string

throw exception if there is a problem